OpenRA.Mods.Common/Widgets/Logic/Ingame/Hotkeys/PauseHotkeyLogic.cs

```diff
@@ -30,8 +30,8 @@ public PauseHotkeyLogic(Widget widget, ModData modData, World world, Dictionary<
 
 		protected override bool OnHotkeyActivated(KeyInput e)
 		{
-			// Disable pausing for spectators and defeated players unless they are the game host
-			if (!Game.IsHost && (world.LocalPlayer == null || world.LocalPlayer.WinState == WinState.Lost))
+			// Disable pausing for spectators and players unless they are the game host.
+			if (!Game.IsHost && (world.LocalPlayer == null || world.LocalPlayer.WinState != WinState.Undefined))
```

Comment:
This looks like it would still allow the host to unpause a game after they have won.